content/blog/tikv-3.0ga-release-notes.md

```diff
@@ -0,0 +1,164 @@
+---
+title: TiKV 3.0 GA Release & Benchmarks
+date: 2019-07-08
+---
+
+Today we're proud to announce the general availability of TiKV 3.0! Before release, TiKV 3.0 underwent a rigorous testing regime, including the [official Jepsen test](https://www.pingcap.com/blog/tidb-passes-jepsen-test-for-snapshot-isolation-and-single-key-linearizability/) with TiDB.
+
+In this version, we tackled many problems of stability at massive scale. Whether spanning hundreds of nodes or storing over a trillion key-value pairs, we've seen our users put TiKV to the test in serious, real-world, production scenarios. In 3.0, we've applied the lessons learned from these deployments to bring a host of new features that can better support users' growing demands.
+
+## Steady at scale
+
+For the 3.0 release, we've improved TiKV by:
+
+* **Optimizing the Raft heartbeat mechanism.** With the [hibernate region](https://github.com/tikv/rfcs/blob/master/text/2019-03-04-hibernate-raft.md) feature, TiKV now adjusts the heartbeat frequency according to region activity. That means you'll see less CPU time and network traffic from idle regions.
+
+* **Distributing Garbage Collection.** The introduction of a distributed garbage collector improves performance on large scale clusters dramatically, leading to better stability through more consistent performance.
```

Comment:
hyperlinking "improves performance on large scale clusters" to evidence would be A+

Reply:
I don't have that kind of data public at the moment. :( There are some tests in our internal logbooks by @MyonKeminta . Can we settle for A- instead? :)

Reply:
I'd settle with A-. Hopefully around next release, we could have comprehensive and formal benchmark data before we publish a release.